lang/fr_FR.UTF-8/LC_MESSAGES/messages.po

```diff
@@ -11808,7 +11808,7 @@ msgstr "PID"
 
 #: centreon-web/www/include/configuration/configServers/listServers.php:117
 msgid "Start time"
-msgstr "Heure de démarrage du programme"
+msgstr "Temps de début"
```

Comment:
```suggestion
msgstr "Heure de début"
```

Reply:
Then I guess we should update the others to stay consistent?